alchemy-web/src/content/docs/providers/aws/ses.md

```diff
@@ -49,3 +49,37 @@ const emailConfig = await SES("tracking-config", {
   },
 });
 ```
+
+## Complete Example with Domain Verification
+
+For a complete setup including domain verification with DNS automation:
+
+```ts
+import { SESVerification } from "alchemy/aws";
+
+// Automate domain verification using Cloudflare DNS
+const verification = await SESVerification("example.com-verification", {
+  domain: "example.com",
+  enableDkim: true,
+  enableReceiving: true,
+  apiToken: process.env.CLOUDFLARE_API_TOKEN,
+});
+
+// Create SES configuration set
+const configSet = await SES("production-config", {
+  configurationSetName: "production-emails",
+  sendingOptions: {
+    SendingEnabled: true,
+  },
+  trackingOptions: {
+    CustomRedirectDomain: "click.example.com",
+  },
+});
```

Comment:
this is not the desired usage. the user should only instantaite `SES` and pass the properties to it

Reply:
```ts
await SES("production-config", {
  configurationSetName: "production-emails",
  sendingOptions: {
    SendingEnabled: true,
  },
  trackingOptions: {
    CustomRedirectDomain: "click.example.com",
  },
  domain: "example.com",
  enableDkim: true,
  enableReceiving: true,
  apiToken: alchemy.secret(process.env.CLOUDFLARE_API_TOKEN),
});
```